API/CLI.md

```diff
@@ -15,6 +15,20 @@ The best place to go next is [the Tessel 2 start experience](http://tessel.io/t2
 ## Updating Tessel 2 On-board OS/Firmware
 
 ### How do I know if I need to update my T2?
+t2-cli and t2-firmware are separately versioned.
+
+* For updating t2-cli:
+
+  If the computer running the t2-cli is connected to the internet, the CLI checks whether it is up to date or not. If it is not, the CLI will provide the update instructions using a notifier to the user (as shown below). It is to be updated with a global npm re-install of the CLI.
```

Comment:
"If it is not" leaves "it" ambiguous– the computer? The CLI? Suggested rephrase: "When a CLI command is run on an internet-connected computer, the CLI automatically checks for updates to the `t2-cli` module. If an update is available..."